sdk/ml/azure-ai-ml/azure/ai/ml/constants/_common.py

```diff
@@ -184,6 +184,7 @@
     "will not create a docker container using the image defined in the environment."
 )
 CONNECTION_API_VERSION_KEY = "ApiVersion"
+CONNECTION_DEFAULT_API_VERSION = "default"
```

Comment:
Can we pick an API version that is valid? E.g. in Azure OpneAI example, they set it to be 2023-05-15 now. https://learn.microsoft.com/en-us/azure/ai-services/openai/quickstart?tabs=command-line&pivots=programming-language-python#create-a-new-python-application

Reply:
The type that was automatically set when I created a WS Conn in the UI was "2023-07-01-preview", so I'll set it to that.